### DISCUSSION TITLE
Need to get rid of an image on the screen

### DISCUSSION THREAD

**Discussion by ichinyc** [aclap-dev/video-downloadhelper#504] [Help and Q&A]
### Your question

Today, I just turned on my desktop and I see a image of Incomplete  dwhelper downloaded page and I cannot get rid of it. 
If anyone know how to do it, please give me a step by step instruction. (please see the attached screen shot)
<img width="2552" alt="Screenshot 2024-05-17 at 20 36 41" src="https://github.com/aclap-dev/video-downloadhelper/assets/170150325/374959c1-3782-4979-ac73-7866940b8813">

### Addon & CoApp version and details

Today, I just turned on my desktop and I see a image of Incomplete  dwhelper downloaded page and I cannot get rid of it. 
If anyone know how to do it, please give me a step by step instruction. (please see the attached screen shot)
<img width="2552" alt="Screenshot 2024-05-17 at 20 36 41" src="https://github.com/aclap-dev/video-downloadhelper/assets/170150325/374959c1-3782-4979-ac73-7866940b8813">
<img width="2552" alt="Screenshot 2024-05-17 at 20 36 41" src="https://github.com/aclap-dev/video-downloadhelper/assets/170150325/8efacbfa-6663-4463-969f-120a4e5ee1a9">

**Comment by paulrouget** [comment]
You have to explain better what's going on here.

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  (Paul does.)  I'm just another user.

Normally, you can click on images in a discussion here on GitHub & it will open the image in its own browser window, where you can magnify it.  When you first posted this yesterday, for some reason I couldn't get that separate browser window.  I couldn't see enough detail in your image in order to comment.  But today, it's working.  Must have been a glitch on GitHub.

I have no experience with Mac.  But that image looks a lot to me like you are showing a file manager window on top of your Mac system desktop.  In other words, this has nothing whatsoever to do with VDH.  You appear to have your file manager auto-launched at bootup, & there is some sort of attribute on it that prevents it from closing.  There may also be an option for "always on top."  Like I say, I don't know Mac, so I'm only guessing.

When Paul asks for a better explanation, he means you should have read & followed the instructions that are on the page you filled out to create your problem description.  You're supposed to actually follow the instructions for providing your Add-on & CoApp details, not simply repeat your problem description.  You're also supposed to tell us what you are trying to do with VDH, specifically, the URLs of web pages on which you believe VDH is malfunctioning.

But like I said, this doesn't even look like a VDH problem.  You appear to have your file manager open on the default VDH download directory.  But that is not a VDH usage problem.  You appear to have some sort of system problem that would best be discussed in some forum other than this one.
